operator/v1/tests/etcds.operator.openshift.io/EtcdBackendQuota.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Etcd"
+crdName: etcds.operator.openshift.io
+featureGate: EtcdBackendQuota
+tests:
+  onCreate:
+    - name: Should be able to create with default BackendQuotaGiB
```

Comment:
Make sure to add a case for minimal, where you don't set anything

Reply:
👍🏽 fixed